L1Trigger/L1TMuonBarrel/src/Twinmux_v1/L1ITMuonBarrelPrimitiveProducer.cc

```diff
@@ -37,7 +37,7 @@ class L1ITMuonBarrelPrimitiveProducer  {
 
 public:
   inline L1ITMuonBarrelPrimitiveProducer(std::unique_ptr<MBLTContainer> _mbltContainer);
-  inline ~L1ITMuonBarrelPrimitiveProducer();
+  inline virtual ~L1ITMuonBarrelPrimitiveProducer()  = default;
```

Comment:
Since this was declared already, it means there must be a definition that exists. Please change this to
`inline virtual ~L1ITMuonBarrelPrimitiveProducer();`.